### ISSUE TITLE
Update the SDK installation instructions with PKG_CONFIG_PATH

### ISSUE THREAD

**Issue by volfpeter** [elementary/docs#25]
Hi!

I'm going through the app development [guide](https://elementary.io/docs/code/getting-started#getting-started) and I faced [this issue](https://elementaryos.stackexchange.com/questions/10338/cant-find-gtk-header-file-for-vala-code-compilation) when trying to build the Hello World application. I've posted my solution yesterday and I was wondering whether adding something like my answer to the [Development Libraries](https://elementary.io/docs/code/getting-started#development-libraries) section would be possible.

Thanks for your great work!

**Comment by tintou** [comment]
This shouldn't have been the case as the folder you're adding is part of the default ones in pkgconfig.
Are you on elementary with the required libraries installed?

**Comment by volfpeter** [comment]
Yes, I was on the latest version of elementary and I followed all steps from the guide. I don't remember exactly, but maybe I installed Juno when it was still in beta.

**Comment by lenemter** [comment]
Closing as stale
